api/api.yaml

```diff
@@ -1,61 +1,90 @@
-swagger: "2.0"
+openapi: 3.0.0
 info:
-  title: A Todo list application
-  version: 1.0.0
-
+  version: 'v1'
+  title: 'A simple API introduction '
 schemes:
-- http
-- https
+  http
```

Comment:
Again follow api standards, Look at how responses are suppose to be formatted

Reply:
done